content/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Why are my contributions not showing up on my profile?
-intro: 'Your profile contributions graph is a record of contributions you''ve made to {% data variables.product.product_name %} repositories. Contributions are timestamped according to Coordinated Universal Time (UTC) rather than your local time zone. Contributions are only counted if they meet certain criteria. In some cases, we may need to rebuild your graph in order for contributions to appear.'
+intro: 'Your profile contributions graph is a record of contributions you've made to {% data variables.product.product_name %} repositories. Contributions are timestamped according to Coordinated Universal Time (UTC) rather than your local time zone. Contributions are only counted if they meet certain criteria. In some cases, we may need to rebuild your graph in order for contributions to appear.'
```

Comment:
Your current change *(which is removing a single quote from the intro description)* will break the whole format and it will cause failing the tests. Keep in mind the double single quotes beside each other are created on purpose.

Reply:
Sorry for that. I'm new on github and was just trying how to make commit and pull request .